src/CurrentRouteInterface.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Router;
+
+use Psr\Http\Message\UriInterface;
+
+interface CurrentRouteInterface
```

Comment:
It's not a `route` at all. Can you please rename it to `provider`?

Reply:
Does this mean that `CurrentRoute` also should be renamed to `CurrentRouteProvider` ?

Reply:
It would be great